src/libraries/System.Text.Json/src/System/Text/Json/Nodes/JsonValue.cs

```diff
@@ -102,10 +102,10 @@ internal override void GetPath(List<string> path, JsonNode? child)
         /// <remarks>
         ///   {T} can be the type or base type of the underlying value.
         ///   If the underlying value is a <see cref="JsonElement"/> then {T} can also be the type of any primitive
-        ///   value supported by current <see cref="JsonElement"/>.<br />
-        ///   Specifying <see cref="object"/> will always succeed and return the underlying value as <see cref="object"/>.<br />
-        ///   The underlying value of a <see cref="JsonValue"/> after deserialization is <see cref="JsonElement"/>,
-        ///   otherwise it is the value specified when the <see cref="JsonValue"/> was created.
+        ///   value supported by current <see cref="JsonElement"/>.
+        ///   Specifying the <see cref="object"/> type for {T} will always succeed and return the underlying value as <see cref="object"/>.<br />
+        ///   The underlying value of a <see cref="JsonValue"/> after deserialization is an instance of <see cref="JsonElement"/>,
+        ///   otherwise it's the value specified when the <see cref="JsonValue"/> was created.
         /// </remarks>
         /// <seealso cref="JsonNode.GetValue{TValue}"></seealso>
```

Comment:
Doesn't this also need renaming?
```suggestion
        /// <seealso cref="JsonNode.GetValue{T}"></seealso>
```

Reply:
Yep. Thanks